omx/gstomx.c

```diff
@@ -1277,6 +1275,11 @@ gst_omx_port_acquire_buffer (GstOMXPort * port, GstOMXBuffer ** buf)
   if (timeout != 40 * GST_MSECOND )
     timeout = -2;
 
+  if (retry_limit && num_retries > 5) {
```

Comment:
Consider using something like GST_OMX_NUM_RETRIES_DEFAULT 5, same for 40 * GST_MSECONDS

Reply:
Changed in commit 577747aa1d7afe875676172b923ad778d8ccd06b